manager_contacts/src/Contact.java

```diff
@@ -0,0 +1,11 @@
+public class Contact {
+    String name;
```

Comment:
@valetodo you should add the private access modifier to maintain the encapsulation concept, you should only access to your classes attributes with getters and setters